_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -110,6 +110,20 @@ You will most likely not need to specify any parameters except for `location`. F
    sudo ./bin/opensearch-keystore add s3.client.default.secret_key
    ```
 
+1. (Optional) If you're using custom S3 endpoint (e.g. MinIO), disable AWS EC2 metadata connection:
+
+   ```bash
+   export AWS_EC2_METADATA_DISABLED=true
+   ```
+
+   If you're installing OpenSearch via Helm, put this in your values file:
```

Comment:
Does this 
```suggestion
   If you're installing OpenSearch using Helm, append the following to your values file:
```

Reply:
You're not appending, you're changing an existing parameter with an empty value (extraEnvs: []): https://artifacthub.io/packages/helm/opensearch-project-helm-charts/opensearch?modal=values

Reply:
Got it. I updated the suggestion.